package.json

```diff
@@ -137,7 +137,7 @@
     "gen:docs-watch": "npm run gen:docs -- --watch",
     "build:docs": "node scripts/build.js",
     "lint": "./node_modules/eslint/bin/eslint.js .",
-    "test": "jest && npm run lint",
+    "test": "jest -u && npm run lint",
```

Comment:
Hey @ivanaszuber, just want to double check with you on this one 😃.

My hunch is that if we update the snapshots every time we run the tests we won't get the full benefit of snapshot testing. That is because the snapshots will always be new, and therefore will never fail.

I think we want to generate/update the snapshots only when we introduce a change, and compare against those when testing to catch unintended mutations.

What do you think about it Ivana?

Reply:
oh yeah, for sure @federicoweber , so sorry for forgetting to put this back 🙈 !! I added this in order to update the tests, as there were some breaking changes that caused the build to fail. My thought was to just fix that and then put it back the way it was, so that every build after that will just compare against previous snapshots.

Maybe I was missing something here though, but I believe we had the same issue in the beginning and updating tests during build helper resolve it. Curious if you found another reason why test were failing 🤔 ?

I'll remove the `-u` in the next branch, thanks for pointing this out 🙇‍♀️

Reply:
Got it @ivanaszuber, and no worries at all 😄.

🤔 yeah, I think we have seen the same error happens. My hunch is that: for some reason, the snapshots that are generated in Travis are different from the ones we generated locally. I played with it briefly but couldn't yet determine the root cause, it's definitely a bit surprising 😕.

If we are still seeing this happening I'm happy to spend some time on it to figure it out. Could be fun to pair on this one Ivana 😄.

Reply:
Ah yeah, that's a great idea @federicoweber ! Let's observe how it behaves moving forward and in case it starts acting up again, pairing up on this one and getting to the bottom of it sounds great 😄 🙌